test/Utils.purs

```diff
@@ -2,24 +2,21 @@ module Test.Utils where
 
 import Prelude
 import Data.Either
-import Control.Monad.Eff
-import Control.Monad.Eff.Exception
-import Control.Monad.Eff.Console hiding (error)
+import Effect
+import Effect.Exception
+import Effect.Console hiding (error)
 import Text.Parsing.Parser (ParseError())
 
-type EffT a =
-  Eff (exception :: EXCEPTION, console :: CONSOLE) a
-
-assertEqual :: forall a. Show a => Eq a => a -> a -> EffT Unit
+assertEqual :: forall a. Show a => Eq a => a -> a -> Effect Unit
 assertEqual x y =
   if x == y
     then pure unit
     else throwException $ error $ show x <> " did not equal " <> show y
 
-err :: forall a. String -> EffT a
+err :: forall a. String -> Effect a
```

Comment:
This function now exists in `exceptions` as `Effect.Exception.throw`; would you mind removing this definition and replacing usages with the library version?

Reply:
sure